Master Project/Assets/Scenes/Stirring/spoonScript.cs

```diff
@@ -16,10 +16,28 @@ public class spoonScript : MonoBehaviour {
     //how far the spoon traveled
```

Comment:
So, I can't add a comment to the correct line. But it looks like your spoonScript and stirringTimer classes are following Java conventions (camelCase class names), but this is C# and so it should be following PascalCase class names. Minor point, but something that should be fixed at some point.